### DISCUSSION TITLE
Plugin in Logic and server on PC send Logic in overload

### DISCUSSION THREAD

**Discussion by LucaAghemo** [apohl79/audiogridder#1266] [Help]
Hello, and thank you for your great work!
I installed the plugin on my Mac (i7-8700K, 32 Gb ram, NVME, Monterey) and the server on my laptop (Lenovo i7 9th gen, 16gb ram) both in the same wired network, to use my Waves plugins (original licenses) that don't run anymore on Monterey because they're now incompatible, but still work on windows. (Almost 300€ upgrade to be able to run the SAME PLUGINS I ALREADY PAID AND OWN?! NO WAY!)
I created a new song in Logic, one instrument track, and called an instance of Waves CLA-2 compressor. It works great, but then Logic goes to the limit in CPU use crackling and popping, and then it goes in overload error and stop.
Opening CPU monitor in Logic tells me that the use of Audiogridder suddenly saturate one single CPU thread. So I tried different tracks, knowing that it would be a failure but only to check the behaviour, and every new track with audiogridder, despite what plugin I call on the PC server, skyrocket a new CPU thread.
The server info screen on windows, on the other side, report a CPU usage on the laptop of about 12% maximum.
I didn't tweak any of the settings because, in such a simple situation, I don't expect the need of any variation.
As I said it is a cable network, a simple 1Gb switch and two short cables on the same desk, plus a router that act as a DHCP server.
Am I doing something wrong?
Thanks again, cheers.
Luca

**Comment by EkimmuMaarid** [comment]
I came here because I tried AudioGridder on Windows 10 yesterday, in order to free up processor power, the program obviously does not work.
I confirm that by loading the instrument into the daw, the load fully occurs inside the program, as if I were running the plugin directly, no wrapping effect occurs.
